components/Login.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
```

Comment:
it would be better UX to not only validate after submit... i would again suggest form component from shadcn, you can look up my pr in trip note to see how i did it in login form

Reply:
Honestly, which big portal validate before clicking "submit"? I can't see this kind of validation on Facebook, Instagram, YT, Gmail etc. Everyone validates after typing 2 inputs and then clicking submit

Reply:
i dont know but its a basic UX thing to do client side validation